packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -90,6 +104,9 @@ contract Attestations is IAttestations, Ownable, Initializable, UsingRegistry, R
     // The token with which attestation request fees are paid
     address attestationRequestFeeToken;
 
+    // The address of the key with which this account wants to sign attestations for
+    address attestationKeyAddress;
```

Comment:
Should this be `authorizedAttestor` for consistency with https://github.com/celo-org/celo-monorepo/pull/1177

IMO it would make it more clear that the account address is the attestor if an authorized attestor has not been set